### PR TITLE
Update ClientInfoPanel to make it a Vertical Tab Pane

### PR DESCRIPTION
**Pull request by mrmrinal**
# Update ClientInfoPanel to support buttons instead of TabView

Vertical Button Tabs were created in order to support the new UI which will be housed

## Checklist

- [ ] Have you ensured [`./src/test`](./src/test) is up to date?
- [x] Have you added javadocs?
- [ ] Have you updated User Guide / Dev Guide in [`./docs/`](./docs)?

#115

### CODE REVIEW

**Comment by bharathcs** [comment]
How it looks rn:
image

File: src/main/java/donnafin/ui/ClientInfoPanel.java [hunk 1]
```diff
@@ -65,4 +81,43 @@ private AttributePanel createAttributePanel(Attribute attr) {
...
+    }
+
+    /**
+     * Updates the VBox content to the Client's Liabilities Details
+     */
+    public void changeTabToLiabilities() {
+        refresh();
+    }
+
+
```
Comment (bharathcs) [inline src/main/java/donnafin/ui/ClientInfoPanel.java:122]:
```suggestion
```

File: src/main/resources/view/ClientInfoPanel.fxml [hunk 2]
```diff
@@ -1,32 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (bharathcs) [inline src/main/resources/view/ClientInfoPanel.fxml:1]: However, using the javafxml to set the styling (colour, border colour, border radius, background radius, text colour for the parts of the fxml) is not typical as it makes it hard to change. Do you think its possible to refactor these into the CSS?

**Review by bharathcs — DISMISSED** [review]
I like how you've done up the components right now, planning the functions in that manner is also good design, great work! 

One point about the CSS, please take a look and call me up if you wanna discuss.

**Review by bharathcs — APPROVED** [review]
Great work mrin

**Review by bharathcs — APPROVED** [review]
Great!

**Comment by sheehui** [comment]
some issues with the GitHub reviews mb

**Comment by bharathcs** [comment]
Incorrect FXML syntax caused an issue, this merge broke master as a result. @sheehui and @Bluntsord narrowed the issue down to a couple of lines in the FXML file, and @mrmrinal deployed a quick solution to solve it.
See issue #129 
See fix #128

**Pull request merged**